sdk/core/generic-core/generic/core/_pipeline_client.py

```diff
@@ -0,0 +1,199 @@
+                    SensitiveHeaderCleanupPolicy(**kwargs) if config.redirect_policy else None,
+                ]
+            )
+        else:
+            if isinstance(per_call_policies, Iterable):
+                per_call_policies_list = list(per_call_policies)
+            else:
+                per_call_policies_list = [per_call_policies]
+            per_call_policies_list.extend(policies)
+            policies = per_call_policies_list
+
+            if isinstance(per_retry_policies, Iterable):
+                per_retry_policies_list = list(per_retry_policies)
+            else:
+                per_retry_policies_list = [per_retry_policies]
+            if len(per_retry_policies_list) > 0:
+                index_of_retry = -1
+                for index, policy in enumerate(policies):
+                    if isinstance(policy, RetryPolicy):
+                        index_of_retry = index
+                if index_of_retry == -1:
+                    raise ValueError(
+                        "Failed to add per_retry_policies; no RetryPolicy found in the supplied list of policies. "
+                    )
+                policies_1 = policies[: index_of_retry + 1]
+                policies_2 = policies[index_of_retry + 1 :]
+                policies_1.extend(per_retry_policies_list)
+                policies_1.extend(policies_2)
+                policies = policies_1
+
+        if transport is None:
+            # Use private import for better typing, mypy and pyright don't like PEP562
+            from .pipeline.transport._requests_basic import RequestsTransport
+
+            transport = RequestsTransport(**kwargs)
+
+        return Pipeline(transport, policies)
+
+    def send_request(self, request: HTTPRequestType, **kwargs: Any) -> HTTPResponseType:
+        """Method that runs the network request through the client's chained policies.
+
+        >>> from generic.core.rest import HttpRequest
+        >>> request = HttpRequest('GET', 'http://www.example.com')
+        <HttpRequest [GET], url: 'http://www.example.com'>
+        >>> response = client.send_request(request)
+        <HttpResponse: 200 OK>
+
+        :param request: The network request you want to make. Required.
+        :type request: ~generic.core.rest.HttpRequest
+        :keyword bool stream: Whether the response payload will be streamed. Defaults to False.
+        :return: The response of your network call. Does not do error handling on your response.
+        :rtype: ~generic.core.rest.HttpResponse
+        """
+        stream = kwargs.pop("stream", False)  # want to add default value
+        return_pipeline_response = kwargs.pop("_return_pipeline_response", False)
+        pipeline_response = self._pipeline.run(request, stream=stream, **kwargs)  # pylint: disable=protected-access
+        if return_pipeline_response:
+            return pipeline_response  # type: ignore  # This is a private API we don't want to type in signature
```

Comment:
Where is this used internally? Do we still need it?

Reply:
Was used in the LRO polling stuff. Seeing as this was removed, I went ahead and removed it.